pkg/login/handoverprinter.go

```diff
@@ -0,0 +1,180 @@
+package login
```

Comment:
This class not part of login package , may be under pkg/jira  or pkg/announcement

Reply:
and need to cover with over 50% unit tests